src/MagicString.js

```diff
@@ -349,6 +349,12 @@ export default class MagicString {
 	}
 
 	overwrite(start, end, content, options) {
+		options = options || {};
+		options.overwrite = !options.contentOnly;
```

Comment:
Modifying a user provided options object isn't great DX and unexpected.

Reply:
Sure, PR welcome

Reply:
yes, you're right. I've sent a PR: https://github.com/Rich-Harris/magic-string/pull/227